src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -972,4 +974,53 @@ public void loanDueDateTruncatedOnCheckoutWhenRecallAnywhereInQueue() {
       String loanDueDate = loan.getJson().getString("dueDate");
       assertThat(loanDueDate, is(truncatedLoanDate.toString()));
   }
+  @Test
+  public void loanDateTruncatedWhenRecalledItemItemHasBeenPreviouslyRecalledandRenewed() {
```

Comment:
Does this test demonstrate that when a loan is renewed, the due is truncated due to the presence of a recall request even if it the due date had been affected already by a previous recall request?